pyfritzhome/devicetypes/fritzhomedevicelightbulb.py

```diff
@@ -65,10 +67,16 @@ def _update_lightbulb_from_node(self, node):
             self.saturation = self.get_node_value_as_int(colorcontrol_element,
                                                          "saturation")
 
+            self.unmapped_hue = self.get_node_value_as_int(colorcontrol_element, "unmapped_hue")
+
+            self.unmapped_saturation = self.get_node_value_as_int(colorcontrol_element,
+                                                         "unmapped_saturation")
```

Comment:
In my Tests hue/unmapped_hue and saturation/unmapped_saturation were always equal, so I'm not sure if we should keep them 😕.

Reply:
Unfortunatly I have no fritz!dect 500. So I cannot say anything about that.